web/src/korrel8r/log.ts

```diff
@@ -19,72 +24,40 @@ export class LogNode extends Korrel8rNode {
     this.logClass = logClass;
   }
 
+  // There are 2 types of URL: pod logs, and log search.
   static fromURL(url: string): Korrel8rNode {
-    if (!url.startsWith('monitoring/logs'))
-      throw new NodeError('Expected url to start with monitoring/logs');
-    const urlQuerySegment = url.split('?').at(1);
-    if (!urlQuerySegment) throw new NodeError('Expected URL to contain query parameters');
-
-    const urlSearchParams = new URLSearchParams(urlQuerySegment);
-    if (urlSearchParams.size === 0)
-      throw new NodeError('Expected more than 0 relevant query parameters');
-
-    let urlQueryString = urlSearchParams.get('q');
-    if (!urlQueryString) throw new NodeError('Expected more than 0 relevant query parameters');
-
-    let logClass = urlSearchParams.get('tenant');
-    if (!logClass) {
-      const logClassRegex = '{[^}]*log_type(=~*)"([^"]+)"}';
-
-      logClass = urlQueryString.match(new RegExp(logClassRegex))?.at(2);
-      if (!logClass) throw new NodeError('Expected query to contain log class');
-    }
-    if (urlQueryString.endsWith('|json')) {
-      urlQueryString = urlQueryString.slice(0, -5);
+    // First check for aggregated pod logs URL
```

Comment:
By losing the ability to round trip the logs when starting on an `aggregated-logs` URL if the "home" node is selected while still on the `aggregated-logs` URL it will take you to the `monitoring/logs` page. I think thats fine for now but what do you think? Maybe we can prevent navigation to selected nodes in the future.

Reply:
+1 this is an issue for several domains where there are "individual" and "search" flavours of URL.
korrel8r only generates "search" style queries for now. However in cases where it executes the query and gets a single result it could conceivably create an "instance" query - something to consider in future.